File: pgm/ABC_Dart/ABC_418_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  stdin.readLineSync()!;
  final String S = stdin.readLineSync()!;
  final String tea = 'tea';
  final String Yes = 'Yes';
  final String No = 'No';
  print(S.endsWith(tea) ? Yes : No);
  exit(0);
}

File: pgm/ABC_PHP/ABC_418_A.php
<?php
$stream = fopen('php://stdin', 'r');
fgets($stream);
$line = fgets($stream);
fclose($stream);
const Yes = 'Yes';
const No = 'No';
const tea = 'tea';
$S = trim($line);
echo (str_ends_with($S, tea) ? Yes : No) . PHP_EOL;
exit(0);
